### ISSUE TITLE
Various string issues

### ISSUE THREAD

**Issue by pryds** [goatpig/BitcoinArmory#77]
The following two strings from qtdialogs.py are postfixed with "cid" in Transifex, which may not be what was intended:

```
if not computedWltID == expectedWltID:
   QMessageBox.critical(parent, tr('Bad Backup!'), tr("""
      If you are sure that you entered the backup information
      correctly, then it is <b>highly recommended you stop using
      this wallet!</b>  If this wallet currently holds any funds,
      you should move the funds to a wallet that <u>does</u>
      have a working backup.
      <br><br>
      Computed wallet ID: %(cid)s <br>
      Expected wallet ID: %(eid)s <br><br>
      Is it possible that you loaded a different backup than the
      one you just made? """ % { 'cid' : computedWltID, 'eid' : expectedWltID }), \
      QMessageBox.Ok)
else:
   MsgBoxCustom(MSGBOX.Good, tr('Backup is Good!'), tr("""
      Your backup works!
      <br><br>
      The wallet ID computed from the data you entered matches
      the expected ID.  This confirms that the backup produces
      the same sequence of private keys as the original wallet!
      <br><br>
      Computed wallet ID: %(cid)s <br>
      Expected wallet ID: %(eid)s <br>
      <br>
      """ % { 'cid' : computedWltID, 'eid' : expectedWltID }))
```

**Comment by pryds** [comment]
Also, notice the following string:

    QMessageBox.information(self, tr("Broadcast!"), tr("""
       Your transaction was successfully sent to the local Bitcoin
       Core instance, though there is no guarantees that it was
       forwarded to the rest of the network.   On testnet, just about
       every valid transaction will successfully propagate.  On the
       main Bitcoin network, this will fail unless it was a standard
       transaction type.

       The transaction 
       had the following hash:
       <br><br>
       %(hexhash)s
       <br><br>
       If this is test
       You can check whether it was seen by other nodes on the network
       blockchain.info (this link will not work on testnet):
       <br><br>
       <a href="%(link)s">%(disp)s</a>""") % \
       { 'hexhash' : hexhash, 'link' : linkToExplorer, 'disp' : dispToExplorer}, QMessageBox.Ok)

It seems that the line "If this is test" is superfluous?

**Comment by pryds** [comment]
This string is from ArmoryQT.py:

    <b>To maximize your security, the Bitcoin engine is downloading
    and verifying the global transaction ledger.  <u>This will take
    several hours, but only needs to be done once</u>!</b>  It is
    usually best to leave it running over night for this
    initialization process.  Subsequent loads will only take a few
    minutes.
    <br><br>
    <b>Please Note:</b> Between Armory and the underlying Bitcoin
    engine, you need to have 40-50 GB of spare disk space available
    to hold the global transaction history.
    <br><br>
    While you wait, you can manage your wallets.  Make new wallets,
    make digital or paper backups, create Bitcoin addresses to receive
    payments,
    sign messages, and/or import private keys.  You will always
    receive Bitcoin payments regardless of whether you are online,
    but you will have to verify that payment through another service
    until Armory is finished this initialization.

The "40-50" estimation seems to be very outdated (e.g. at this moment my own ~/.bitcoin/blocks folder takes up more than 90 GB), and it probably will be again soon, even if adjusted. Maybe the numbers should not be a part of the string directly, so that it can be adjusted without translators having to adjust their translations?

**Comment by pryds** [comment]
From qtdialogs.py:

    Please make sure that any printed backups you create  (excluding any "ID" lines) have <b>nine
    columns</b> of four letters
    each.
    If you just made a paper backup, it is important that you test it
    to make sure that it was printed or copied correctly.  Most importantly,

This string ends rather abruptly.

**Comment by achow101** [comment]
This can be closed. String issues were fixed in #89
